tests/test_sanity.py

```diff
@@ -241,3 +243,33 @@ def test_from_container_spawned_process(
         assert_collapsed(collapsed)
 
     assert b"Profiling spawned process" in container.logs()
+
+
+@pytest.mark.parametrize("in_container", [True])
+@pytest.mark.parametrize("profiler_type", ["attach-maps"])
+@pytest.mark.parametrize("runtime", ["nodejs"])
+@pytest.mark.parametrize("application_image_tag", ["without-flags"])
+def test_container_name_when_stopped(
```

Comment:
I'd add some docstring explaining the test and linking to the GH ticket. Something short like here:
```
@pytest.mark.parametrize("in_container", [True])
def test_java_jattach_async_profiler_log_output(
    application_pid: int,
    caplog: LogCaptureFixture,
    profiler_state: ProfilerState,
) -> None:
    """
    Tests that AP log is collected and logged in gProfiler's log.
    """
    ...
```